apps/doji-frontend/src/components/ExpertService/TagInput.tsx

```diff
@@ -40,23 +22,33 @@ export default function TagsInput(props: TagInputProps) {
         }}
         filterSelectedOptions={true}
         multiple
-        options={friendList}
-        getOptionLabel={(option) => option.text}
+        options={props.friendList}
+        getOptionLabel={(option) => option.value}
         renderTags={(value, getTagProps) =>
           value.map((option, index) => (
             <Chip
               key={option.value}
               variant="outlined"
-              label={option.text}
-              avatar={<Avatar></Avatar>}
+              label={option.value}
+              avatar={<Avatar src={option.profileImageURL} alt={option.value}></Avatar>}
```

Comment:
what if user has no profileImageUrl?

Reply:
it will show the default image instead, I cant figure out how to show the first character

Reply:
I fixed it